src/com/chrisfolger/needsmoredojo/core/amd/importing/ImportCreator.java

```diff
@@ -37,7 +37,7 @@ public void createImport(String module, String quoteCharacter, String parameter,
             }
         }
 
-        if(imports.getChildren().length == 0)
+        if(imports.getExpressions().length == 0)
```

Comment:
What is the benefit of using `getExpressions()` over `getChildren()` here?

Reply:
There was a problem that when adding an import (Ctrl+Shift+O, 2), the import was added in the wrong location, usually outside of the JavaScript import array. Using `getExpressions()` fixed this issue.